cmd/helm/template.go

```diff
@@ -206,12 +207,7 @@ func newTemplateCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 }
 
 func isTestHook(h *release.Hook) bool {
-	for _, e := range h.Events {
-		if e == release.HookTest {
-			return true
-		}
-	}
-	return false
+	return slices.Contains(h.Events,release.HookTest)
```

Comment:
```suggestion
	return slices.Contains(h.Events, release.HookTest)
```

need to turn on go format in your editor :)

Reply:
Sorry for my carelessness.

I have modified it. Please review again.